sources/api/datastore/src/lib.rs

```diff
@@ -147,7 +147,14 @@ pub trait DataStore {
         S: AsRef<str>,
     {
         for (key, value) in pairs {
-            trace!("Setting data key {}", key.name());
+            match committed {
+                Committed::Live => {
+                    info!("Setting data key {}", key.name());
+                }
+                _ => {
+                    trace!("Updating data key {}", key.name());
+                }
```

Comment:
It seems like the `Committed` enum is used to describe the state of a transaction rather than whether or not the write is a set/update. Is this right?

Reply:
You probably know our datastore operations better than I do. Are there any cases where we would be committing something that would not be a set/update?